text.js

```diff
@@ -178,7 +180,15 @@ FastTextDecoder.prototype.decode = function(buffer, options={stream: false}) {
     }
   }
 
-  return String.fromCharCode.apply(null, out);
+  // itterate over the out buffer
```

Comment:
"iterate"

the comment should also explain why it's chunking things up (to avoid stack recursion)